site_scons/site_tools/roc/commands.py

```diff
@@ -103,6 +103,33 @@ def Ragel(env, source):
 
     return [env.Object(target)]
 
+def PkgConfig(env, build_dir, prefix, libdir, filename, version, dependencies):
+    output_dir = env.Dir(build_dir).path
+    target = os.path.join(output_dir, filename)
+    src = """prefix=%(prefix)s
+exec_prefix=${prefix}
+libdir=%(libdir)s
+includedir=${prefix}/include
+
+Name: libroc
+Requires: %(dependencies)s
+Version: %(version)s
+Description: Real-time audio streaming over the network.
+URL: https://roc-project.github.io
+
+Libs: -L${libdir} -lroc -lrt -lpthread -lnsl -ldl -lm
```

Comment:
Hard-coding this list:

```
-lrt -lpthread -lnsl -ldl -lm
```

is not portable.

`-lrt` and `-ldl` are not available on all systems, for example.

Instead, let's find the places in SConstruct where they're added, for example:

```
if platform in ['linux']:
    env.AppendUnique(LIBS=['rt', 'dl', 'm'])
```

And when they're added, let's also add them to some array like `env['PKG_CONFIG_LIBS']`. We can add a new method for this, like `env.AddPkgConfigLib('rt')`.